code/game/machinery/cloning.dm

```diff
@@ -389,8 +389,21 @@
 				user.visible_message("[user] secures [src] to the floor.", "You secure [src] to the floor.")
 			else
 				user.visible_message("[user] unsecures [src] from the floor.", "You unsecure [src] from the floor.")
+	else if(istype(W, /obj/item/device/multitool))
+		var/obj/item/device/multitool/M = W
+		M.buffer = src

```

Comment:
I don't think this will work quite right anymore. We have /vg/'s multitool buffer system now.